### PR TITLE
exercises completed

### PR DESCRIPTION
**Pull request by andreaseminidis**
Please let me know if you have any questions.

### CODE REVIEW

File: src/main/scala/helpers/LocationHelpers.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    *
+    * Calculates the distance (estimation) between 2 GPS coordinates.
+    *
+    * @param lat1 latitude 1
+    * @param lon1 longitude 1
+    * @param lat2 latitude 2
+    * @param lon2 longitude 2
+    * @return distance between 2 GPS coordinates
+    */
+  def coordDistance(lat1: Double, lon1: Double)(lat2: Double, lon2: Double): Double = {
```
Comment (dsmileym4) [inline src/main/scala/helpers/LocationHelpers.scala]: What would an alternative be to Haversine?

File: src/test/helpers/LocationHelpersTest.scala [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package helpers
+
+import org.scalatest._
+
+class LocationHelpersTest extends FlatSpec with Matchers with LocationHelpers {
+
+  "A Location Helper" should "compute distance between 2 coordinates" in {
+
+    coordDistance(0.0, 0.0)(0.0,0.0) should be (0.0)
```
Comment (dsmileym4) [inline src/test/helpers/LocationHelpersTest.scala]: How would you improve this test? How would you approach building a test suite for the entire applications?

**Comment by dsmileym4** [comment]
Overall this looks great! Good work.

File: src/main/scala/helpers/LocationHelpers.scala (hunk 1 above)
Comment (andreaseminidis) [reply src/main/scala/helpers/LocationHelpers.scala]: Vincenty can be used to be more accurate. On the other hand, an equirectangular approximation is faster to computer and simpler to implement, but it is less accurate, especially for larger distances.

**Pull request closed without merging**